src/main/java/com/smartbear/readyapi/plugin/git/GitCommandHelper.java

```diff
@@ -375,4 +380,57 @@ private boolean canMerge(final Git git, final MergeStrategy mergeStrategy) {
 
         return true;
     }
+
+    public String getCurrentBranch(final Git git) {
+        try {
+            return git.getRepository().getFullBranch();
+        } catch (IOException e) {
+            throw new VcsIntegrationException("Can get current branch", e);
```

Comment:
As I see this error message is incorrect.

Reply:
Fixed